app/helpers/patients_helper.rb

```diff
@@ -14,6 +14,10 @@ def race_ethnicity_options
      'Native American', 'Mixed Race/Ethnicity', 'Other']
   end
 
+  def language
+    [nil, 'English', 'Spanish', 'French', 'Korean']
```

Comment:
Two considerations I'll throw in the mix:
1) Is English the best default for all funds that will deploy the app?
2) Is it useful in the data analytics to use as a proxy for whether the question is being asked and answered as English vs the question wasn't asked?

I'm fine with either solution, I'll wait to hear back from @alisonnjones before I make the change